### DISCUSSION TITLE
Can't sell items from the crate in deconstructor

### DISCUSSION THREAD

**Discussion by Kokhirus** [FakeFishGames/Barotrauma#14051] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Kind of a small take, but you cannot sell items from the crate that is located in a deconstructor

### Reproduction steps

1. Put an item inside a crate
2. Put the crate in a deconstructor
3. Try to sell the item from a submarine

### Bug prevalence

Just once

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
I believe this is intentional, items from fabricators and deconstructors don't show up in the store interface to prevent accidentally selling materials that weren't meant to be sold (like let's say if someone is currently processing materials for something they want to fabricate).

**Reply by Kokhirus** [reply]
Oh, ok, that makes sense then. I've got slightly similar question about containers in hands: you cannot craft from its contents if the container is not in the fabricator itself. Maybe it is intentional as well, so cannot tell for sure if this is a bad thing.
